core/iwasm/libraries/lib-socket/test/build.sh

```diff
@@ -0,0 +1,14 @@
+
+# Copyright (C) 2019 Intel Corporation.  All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+CC="${CC:=/opt/wasi-sdk/bin/clang}"
+$CC \
```

Comment:
to avoid duplication you can iterate through array instead.

Reply:
done